rust/counter/README.md

```diff
@@ -0,0 +1,90 @@
+# Counter
+
+This is a Rust / Yew port of the Mokoto Counter example. 
+
+This example demonstrates a counter application. It uses an orthogonally
+persistent `counter` variable to store an arbitrary precision natural number
+that represents the current value of the counter.
+
+By using the Motoko keyword `stable` when declaring the `counter` variable,
+the value of this variable will automatically be preserved whenever your canister code is
+upgraded. Without the `stable` keyword, a variable is deemed `flexible`, and its value
+is reinitialized on every canister upgrade, i.e. whenever new code is deployed to the canister.
+
+To learn more about these features of Motoko, see:
+* https://sdk.dfinity.org/docs/language-guide/motoko.html#_orthogonal_persistence
+* https://sdk.dfinity.org/docs/language-guide/upgrades.html#_declaring_stable_variables
```

Comment:
Old stuff from the Motoko original, should be adapted for Rust.

Reply:
I have rewritten this section in the README attempting to discuss the mechanics of Rust. Please can you confirm that what I have said is technically correct.